### PR TITLE
Ticket details and updation

### PR DESCRIPTION
**Pull request by SWATIK31**
- Implemented the final touchups in ticket creation and improved the ticket updation feature .
- Also implemented a Toast notification on successful updation of ticket.

### CODE REVIEW

File: src/Screens/TicketDetails/TicketDetailModal.jsx [hunk 1]
```diff
@@ -54,46 +56,87 @@ const TicketDetailModal = ({ isOpen, onClose, ticket, onTicketUpdate }) => {
...
               className="modal-input"
               name="status"
               value={editedTicket.status}
               onChange={handleInputChange}
             >
-              <option value="Open">Open</option>
-              <option value="In Progress">In Progress</option>
-              <option value="Not yet started">Not yet started</option>
-              <option value="Merged and closed">Merged and closed</option>
+              <option value="" disabled>Select status</option>
```
Comment (Jha2508) [inline src/Screens/TicketDetails/TicketDetailModal.jsx]: Constants

File: src/Screens/TicketDetails/TicketDetailModal.jsx [hunk 2]
```diff
@@ -54,46 +56,87 @@ const TicketDetailModal = ({ isOpen, onClose, ticket, onTicketUpdate }) => {
...
+              <option value="APPROVED">APPROVED</option>
+              <option value="MERGED">MERGED</option>
             </select>
             <select
               className="modal-input"
               name="priority"
               value={editedTicket.priority}
               onChange={handleInputChange}
             >
+              <option value="" disabled>Select priority</option>
```
Comment (Jha2508) [inline src/Screens/TicketDetails/TicketDetailModal.jsx]: why is the option value empty

File: src/constants/ticketdetailsconstant.js [hunk 3]
```diff
@@ -1,41 +0,0 @@
-
-export const TEXTS = {
-    NAV_HEADING: "New Ticket",
-    DEVELOPER_LABEL: "Developer",
-    STATUS_LABEL: "Ticket Status",
-    PRODUCT_LABEL: "Ticket Product",
-    TYPE_LABEL: "Type",
```
Comment (Jha2508) [inline src/constants/ticketdetailsconstant.js:7]: Why are we deleting this filee?

Comment (SWATIK31) [reply src/constants/ticketdetailsconstant.js:7]: @Jha2508 As I moved all the constants into a single file (constants.js)  for better readability.

**Review by Jha2508 — APPROVED** [review]

**Pull request merged**